### PR TITLE
bugfix: fix function redeclaration in removeunusedservices

### PR DESCRIPTION
**Pull request by stobrien89**
*Issue #, if available:*
#2492

*Description of changes:*
Addresses a conflict with autoload files/imports when script is invoked.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kayw-geek** [comment]
> _Issue #, if available:_ #2492
> 
> _Description of changes:_ Addresses a conflict with autoload files/imports when script is invoked.
> 
> By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

I find the reason for the problem a long time
I already tried your solution, It has solved my problem, thank you!

File: src/Script/Composer/Composer.php [hunk 1]
```diff
@@ -42,7 +39,8 @@ public static function removeUnusedServices(
     public static function buildServiceMapping()
     {
         $serviceMapping = [];
-        $source = Aws\manifest();
+        $manifest = __DIR__ . '/../../data/manifest.json.php';
```
Comment (kayw-geek) [inline src/Script/Composer/Composer.php:42]: Do we need the definition of the variable `$mainfest`?

File: src/Script/Composer/Composer.php [hunk 2]
```diff
@@ -42,7 +39,8 @@ public static function removeUnusedServices(
...
+        $source = include($manifest);
```
Comment (kayw-geek) [inline src/Script/Composer/Composer.php:43]: I think when the `manifest.json.php` file is not found, more should be used `require()`

Comment (stobrien89) [reply src/Script/Composer/Composer.php:43]: Agree that this should throw an error when the file isn't found.  Thanks!

File: src/Script/Composer/Composer.php (hunk 1 above)
Comment (stobrien89) [reply src/Script/Composer/Composer.php:42]: I threw this PR up pretty quickly and (without checking) thought the combination of the two lines might throw us over the PSR-recommended line length.  Looks like it's well below the max recommended line length (63 char).  Nice catch

File: src/Script/Composer/Composer.php [hunk 3]
```diff
@@ -92,7 +89,7 @@ private static function removeServiceDirs(
             }
         }
         $event->getIO()->write(
-            "Removed $deleteCount AWS services"
+            "Removed $deleteCount AWS service" . ($deleteCount === 1 ? '' : 's')
```
Comment (SamRemis) [inline src/Script/Composer/Composer.php:92]: I think this is simpler, but [ngettext](https://www.php.net/manual/en/function.ngettext.php) is a thing in PHP.  I like this better, but I guess ngettext is more idiomatic?

**Review by SamRemis — APPROVED** [review]

File: src/Script/Composer/Composer.php (hunk 3 above)
Comment (kayw-geek) [reply src/Script/Composer/Composer.php:92]: The `$deleteCount` probably will be `0`

**Pull request merged**